tests/smoke/multi_cluster_deploy_test.go

```diff
@@ -0,0 +1,217 @@
+package smoke
+
+import (
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/rook/rook/tests/framework/clients"
+	"github.com/rook/rook/tests/framework/enums"
+	"github.com/rook/rook/tests/framework/installer"
+	"github.com/rook/rook/tests/framework/utils"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	utilversion "k8s.io/kubernetes/pkg/util/version"
+)
+
+func TestInstallingMultipleRookCluster(t *testing.T) {
+	suite.Run(t, new(MultiRookClusterDeploySuite))
+}
+
+type MultiRookClusterDeploySuite struct {
+	suite.Suite
+	helper1     *clients.TestClient
+	helper2     *clients.TestClient
+	k8sh        *utils.K8sHelper
+	installer   *installer.InstallHelper
+	installData *installer.InstallData
+}
+
+//Deploy Multiple Rook clusters
+func (mrc *MultiRookClusterDeploySuite) SetupSuite() {
+	kh, err := utils.CreateK8sHelper()
+	require.NoError(mrc.T(), err)
+
+	mrc.k8sh = kh
+	mrc.installer = installer.NewK8sRookhelper(kh.Clientset)
+	mrc.installData = installer.NewK8sInstallData()
+
+	err = mrc.installer.CreateK8sRookOperator()
+	require.NoError(mrc.T(), err)
+
+	require.True(mrc.T(), kh.IsPodInExpectedState("rook-operator", "default", "Running"),
+		"Make sure rook-operator is in running state")
+
+	time.Sleep(10 * time.Second)
+
+	err = mrc.installer.CreateK8sRookCluster(clusterNamespace1)
```

Comment:
there is a lot of duplication. Could you have a helper that creates and verifies each namespace?